LICENSE-BSD-3-CLAUSE.md

```diff
@@ -0,0 +1,13 @@
+# The 3-Clause BSD License
```

Comment:
Nitpicky. The best name for this file is probably `LICENSE-BSD-3-Clause.md` so we use the capitalization of the SPDX short identifier for the license ID.